cmd/expand.go

```diff
@@ -78,6 +78,7 @@ func runExpandCmd(cmd *cobra.Command, args []string) {
 	if err := deploymentConfig.ExpandConfig(); err != nil {
 		log.Fatal(err)
 	}
+	deploymentConfig.Config.GhpcVersion = GitCommitInfo
```

Comment:
I would like to see a warning that informs the user that any value they set was ignored.

Reply:
In fact, what occurs with this value when ingesting an expanded blueprint?

Reply:
>  a warning 

Done
 
> what occurs with this value when ingesting an expanded blueprint?

This property is intentionally inert  so far.